src/ICS20Transfer.sol

```diff
@@ -23,7 +24,10 @@ using SafeERC20 for IERC20;
  */
 contract ICS20Transfer is IIBCApp, IICS20Transfer, IICS20Errors, Ownable, ReentrancyGuard {
     /// @notice Mapping of non-native denoms to their respective IBCERC20 contracts created here
-    mapping(string denom => IBCERC20 ibcERC20Contract) private _foreignDenomContracts;
+    mapping(string denom => IBCERC20 ibcERC20Contract) private _ibcDenomContracts;
+
+    /// @notice Mapping of channel ID to escrow contract address
+    mapping(string channelId => ICS20Escrow ics20EscrowContract) private _channelEscrowContracts;
```

Comment:
Maybe we should use one escrow contract for every channel @sangier @gjermundgaraba? Just throwing the idea out there because this might make the migrations simpler. However, this is fine for now

Reply:
We decided that we want to have only 1 escrow contract to make migrations simpler